tools/makecsr/README.md

```diff
@@ -50,3 +50,47 @@ Note that the `label` must match between the two configuration files.
 
 For more information, see
 https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=87365053
+
+### Putting it all together, practical GCP example
+Using this with GCP, you should have:
+
+1. A read only libkmsp11-config.yaml file like this
+```
+tokens:
+  - key_ring: projects/my-project/locations/global/keyRings/my-key-ring
+    label: gcp-token
+# Note: This file should be read-only. You can do `chmod -w libkmsp11-config.yaml` after you create it.
+```
+
+2. A crypto11-config.json file like this
+```
+{
+  "Path": "/app/libkmsp11.so",
+  "TokenLabel": "gcp-autograph-token"
+}
+```
+
+3. Be authenticated with GCP. Ex: `gcloud auth login --update-adc`
```

Comment:
(suggestion) I think we need to add a step here about giving the permission to the developer to access and use the key?

Reply:
Yeah, pushed that info to the top. Also revised the points to read better.